packages/core/getBusyTimes.ts

```diff
@@ -39,12 +42,19 @@ export async function getBusyTimes(params: {
         startTime: true,
         endTime: true,
         title: true,
+        eventType: {
+          select: {
+            afterEventBuffer: true,
+          },
+        },
       },
     })
     .then((bookings) =>
-      bookings.map(({ startTime, endTime, title, id }) => ({
-        end: endTime,
+      bookings.map(({ startTime, endTime, title, id, eventType }) => ({
         start: startTime,
+        end: dayjs(endTime)
```

Comment:
wouldn't it make more sense to leave it without buffer times here and add it in getUserAvailability in bufferedBusyTimes?